src/components/home/Home.js

```diff
@@ -39,17 +39,24 @@ function Home() {
     }
 
     async function onAddToDo(e) {
+        setError({
+            toDoName: null,
+            toDoCategory: null,
+        });
         e.preventDefault();
         if (toDoName.length < 1) {
-            setErrors({
+            setError((error) => ({
+                ...error,
                 toDoName: "please fill in a description.",
-            });
-        } else if (toDoCategory === "make a choice" || toDoCategory === null) {
-            setErrors({
+            }));
+        }
+        if (toDoCategory === null) {
+            setError((error) => ({
+                ...error,
                 toDoCategory: "please pick a category.",
-            });
+            }));
         }
-        if (toDoName.length < 1 || toDoCategory === "make a choice" || toDoCategory === null) {
+        if (toDoName.length < 1 || toDoCategory === null) {
```

Comment:
also check what happens if you pick a category, then manually put it back to 'make a choice' -> currently no error shown